src/seedu/addressbook/commands/FindCommand.java

```diff
@@ -16,7 +16,7 @@
 
     public static final String COMMAND_WORD = "find";
 
-    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Finds all persons whose names contain any of "
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Finds all persons whose details (name, number, email) contain any of "
```

Comment:
Looks good to me! You might want to consider allowing search for all other available fields instead of limiting yourself to just name number and email